benchmarks/test_trainer_parity.py

```diff
@@ -64,6 +64,21 @@ def test_pytorch_parity(tmpdir):
     for pl_out, pt_out in zip(lightning_outs, manual_outs):
         np.testing.assert_almost_equal(pl_out, pt_out, 5)
 
+    assert_speed_parity(pl_times, pl_times, num_epochs)
+
+
+def assert_speed_parity(pl_times, pt_times, num_epochs):
```

Comment:
do we really need this two times? Can't we just import the other one?

Reply:
fair, i was just trying to find the threshold number before finalizing pr. will refactor this